lang/java/reef-common/src/main/java/org/apache/reef/io/TcpPortConfigurationProvider.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
```

Comment:
Please use the non-javadoc form for comments and remove the `<p>` tags. See [REEF-315](https://issues.apache.org/jira/browse/REEF-315)